### ISSUE TITLE
The Ads do not appear when the app is downloaded from the store in iOS (also TestFlight)

### ISSUE THREAD

**Issue by ignaciolarranaga** [EddyVerbruggen/nativescript-admob#19]
I can not explain why, but the ads appear correctly on the emulator (tested many times with production and testing) but once I sent the application to the store it does not show the ads (testing through TestFlight not yet released because of the problem).
I sent the application twice so far testing different approaches without success.

Any suggestion ?, it does not affect the app also (like crash) it is just the ads do not appear.

**Comment by ignaciolarranaga** [comment]
Hey guys, some idea ?

**Comment by ignaciolarranaga** [comment]
I got more detail about the problem, it is throwing:

```
There was an error creating the AdMob banner: ReferenceError: Metadata for \"GoogleMobileAds.kGADAdSizeSmartBannerPortrait\" found but symbol not available at runtime.
```

**Comment by ignaciolarranaga** [comment]
Just found #14 and seems to be the same problem that is still happening.

**Comment by EddyVerbruggen** [comment]
Anyone else has this issue? It should happen to everybody since forever..

**Comment by ignaciolarranaga** [comment]
Might it be just for the Smart Banner ?

**Comment by EddyVerbruggen** [comment]
Don't know at this point. I'm mobile atm and can't check the file in platforms/ios that's responsible for exposing the values for release builds. 

Would be VERY interested to learn from anyone that doesn't have this problem for any banner type with release builds..

**Comment by ignaciolarranaga** [comment]
I'm now testing with other sizes, by chance do you have how to construct the size manually in case it does not work ?, will update in a few minutes once I upload the app and download it again for validation.

**Comment by ignaciolarranaga** [comment]
I'm afraid it is the same with just Banner instead of SmartBanner:

`"There was an error creating the AdMob banner: ReferenceError: Metadata for \"GoogleMobileAds.kGADAdSizeBanner\" found but symbol not available at runtime."`

Will try setting the size by hand as you suggested in #14 

If anyone has some additional suggestion will be welcome.

Just to comment I'm also using other libraries that include google things, here is the complete list, because it might be result of a colision (I was just thinking on it):
```
"dependencies": {
    "@angular/common": "2.4.3",
    "@angular/compiler": "2.4.3",
    "@angular/core": "2.4.3",
    "@angular/forms": "2.4.3",
    "@angular/http": "2.4.3",
    "@angular/platform-browser": "2.4.3",
    "@angular/platform-browser-dynamic": "2.4.3",
    "@angular/router": "3.4.3",
    "buffer": "4.6.0",
    "google-libphonenumber": "2.0.8",
    "nativescript-admob": "~1.3.0",
    "nativescript-angular": "1.4.0",
    "nativescript-facebook-login": "^0.2.0",
    "nativescript-fcuuid": "1.0.2",
    "nativescript-fresco": "^1.0.15",
    "nativescript-google-analytics": "~0.3.5",
    "nativescript-intl": "0.0.10",
    "nativescript-iqkeyboardmanager": "^1.0.1",
    "nativescript-phone": "1.2.3",
    "nativescript-push-notifications": "0.0.19",
    "nativescript-social-share": "^1.3.2",
    "nativescript-telerik-ui": "~1.6.1",
    "nativescript-theme-core": "~1.0.2",
    "ng2-translate": "4.0.0",
    "reflect-metadata": "~0.1.8",
    "rxjs": "~5.0.1",
    "tns-core-modules": "2.5.0",
    "validator": "5.7.0"
  },
```

**Comment by gla5001** [comment]
it happens with any banner ad. The interstitial ads work fine. I ended up redesigning my app a little for iOS to make this work.

**Comment by ignaciolarranaga** [comment]
Just to update that I tried to replace the constant by the actual value (ends up being just an string), but it does not work (it works on the simulator but it throws the same problem when run after downloading from TestFlight).

I'm afraid this error turns the plugin unusable on iOS unless we found a workaround.

**Comment by EddyVerbruggen** [comment]
Hi @ignaciolarranaga, thanks for your efforts! Because of your report I've been looking at this issue today as well and may have good news shortly!

**Comment by EddyVerbruggen** [comment]
Yes! Got it behaving correctly on TestFlight. Will push a new release in about an hour.

**Comment by EddyVerbruggen** [comment]
OK, found the correct way to replace those symbols by constants. Released with 1.3.1.

**Comment by EddyVerbruggen** [comment]
@gla5001 Btw, Interstitials indeed didn't have this problem.

**Comment by ignaciolarranaga** [comment]
@EddyVerbruggen just to confirm it is working except for the SMART_BANNER (it does not even appear on the simulator). Thank you so much for your support!

**Comment by EddyVerbruggen** [comment]
Thanks for testing that! I have an idea why smart banners are not shown. 
Expect a fix tomorrow.

**Comment by EddyVerbruggen** [comment]
Smart banners have been fixed in 1.3.2 🎉
